packages/reports/addon/components/common-actions/schedule.ts

```diff
@@ -134,88 +144,105 @@ export default class ScheduleActionComponent extends Component<Args> {
       return true;
     }
 
-    return !(this.localDeliveryRule?.hasDirtyAttributes && this.isRuleValid);
+    return !(this.localDeliveryRule?.hasDirtyAttributes && this.areRulesValid);
```

Comment:
Did we move do individual saves? If so, do we need to check if all are valid to save one rule?

Reply:
We didnt move to individual saves